$NetBSD: patch-xpcom_reflect_xptcall_md_unix_moz.build,v 1.11 2018/11/07 12:55:11 martin Exp $

Make NetBSD/sparc64 use the same xptcall bindings as all other sparc64 ports

--- ./xpcom/reflect/xptcall/md/unix/moz.build.orig	2018-10-31 01:08:15.000000000 +0100
+++ ./xpcom/reflect/xptcall/md/unix/moz.build	2018-11-07 10:29:36.234212381 +0100
@@ -240,7 +240,7 @@
         'xptcstubs_sparc_openbsd.cpp',
     ]
 
-if CONFIG['OS_ARCH'] in ('OpenBSD', 'FreeBSD', 'Linux') and CONFIG['CPU_ARCH'] == 'sparc64':
+if CONFIG['OS_ARCH'] in ('OpenBSD', 'FreeBSD', 'NetBSD', 'Linux') and CONFIG['CPU_ARCH'] == 'sparc64':
     SOURCES += [
         'xptcinvoke_asm_sparc64_openbsd.s',
         'xptcinvoke_sparc64_openbsd.cpp',
